### ISSUE TITLE
Task #8: Split data into Training and Validation sets

### ISSUE THREAD

**Issue by grayson-a-nelson** [Break-Through-Tech/Oregon-State-University-1A-forecasting-risk-of-harmful-algal-blooms-in-coastal-marine-ecosystems#10]
Separate the dataset into a training set used to build and experiment with models and a validation set to test the model's performance on unseen data.
